editor/src/components/canvas/controls/select-mode/simple-outline-control.tsx

```diff
@@ -28,26 +28,32 @@ export const MultiSelectOutlineControl = React.memo<MultiSelectOutlineControlPro
     <CanvasOffsetWrapper>
       {[
         <OutlineControl
+          testId={`multiselect-outline`}
           key='multiselect-outline'
           targets={localSelectedElements}
           color='multiselect-bounds'
           outlineStyle='solid'
         />,
-        ...localSelectedElements.map((path) => (
-          <OutlineControl
-            key={EP.toString(path)}
-            targets={[path]}
-            color='primary'
-            outlineStyle='solid'
-          />
-        )),
+        ...localSelectedElements.map((path) => {
+          const varSafeId = `multiselect-element-outline-${EP.toVarSafeComponentId(path)}`
```

Comment:
I don't believe there's any need for this to be "varSafe". I actually don't think we need `toVarSafeComponentId` at all anymore, as that's left over from the old old Utopia

Reply:
ORLY?

Reply:
Removed in b26e6a405.

Reply:
Yeah, it was used for creating variable names that were valid in JS. I don't recall specifically why.